lib/linguist/languages.yml

```diff
@@ -3609,6 +3609,15 @@ NetLogo:
   codemirror_mode: commonlisp
   codemirror_mime_type: text/x-common-lisp
   language_id: 246
+Netlify Redirects:
```

Comment:
Hrm, perhaps we should name this `Redirect Rules` instead? The file-format looks very generic, and I imagine the syntax could be repurposed by other routers and web-servers (so omitting the obvious Netlify connection might encourage reuse).

`.gitignore` files were named _"Ignore List"_ for this very same reason—it enabled us to add several other mostly-identical formats in a clean, sane fashion.